modules/dnn/src/tensorflow/tf_graph_simplifier.cpp

```diff
@@ -719,13 +729,20 @@ class PReLUSubgraph : public TFSubgraph
     {
         if (!negativeScales)
         {
-            Mat scalesRef = getTensorContentRefUnaligned(inputNodes[1]->attr().at("value").tensor());
-            Mat scales = scalesRef.clone() * -1;
+            Mat scalesRef = getTensorContentRef_(inputNodes[1]->attr().at("value").tensor());
             // FIXME: This breaks the const guarantees of tensor() by writing to scalesRef and is likely to perform an
             //        unaligned write (hence the separate multiplication above to have this work on ARM).
```

Comment:
Comment's reference to multiplication (in brackets) is outdated with this commit